### DISCUSSION TITLE
我是艾莲的狗

### DISCUSSION THREAD

**Discussion by biliqide** [share121/inter-knot#1351] [其他]
艾莲喜欢什么味道的棒棒糖？在线求答![Screenshot_2024-08-11-13-27-58-89_2332cb9b27b851b548ba47a91682926c](https://github.com/user-attachments/assets/42208929-6ba5-4ddf-9366-fb00d4902030)

**Comment by sishui-buyin** [comment]
你起开我才是 🤬😡😡😡
